docs/content/_navigation.json

```diff
@@ -242,19 +242,19 @@
         ]
       },
       {
-        "title": "Dagit UI & GraphQL API",
+        "title": "Dagster webserver & UI",
```

Comment:
>should we call it Dagster UI & GraphQL API?

IMO no, since "Dagster webserver" and "Dagster UI" are referred to many times in the docs, whereas the GraphQL API is less common referenced.